templates/base.html

```diff
@@ -121,11 +54,34 @@
     <div class="u-hide" id="contact-form-container" data-form-location="/get-in-touch" data-form-id="1337" data-lp-id="2313" data-return-url="http://juju.is/thank-you" data-lp-url=""></div>
     <script src="{{ versioned_static('js/dynamic-contact-form.js') }}"></script>
 
-    <script src="/static/js/build/global-nav/global-nav.js"></script>
+    <script src="/static/js/navigation.js"></script>
```

Comment:
You will need to transpile this js otherwise you will hit issues on older browsers

Reply:
Hmm...considering the syntax of this file, I think it should be ok even for old browsers https://github.com/canonical-web-and-design/juju.is/pull/188/files#diff-7b2c4db56e5b529368acd62d2b00b0a52cd5c45f1a093bf1ed4c4d69aeacfa51R7